src/views/gateway_configuration/GatewayConfigFormEdit.js

```diff
@@ -26,26 +26,53 @@ export const GatewayConfigFormEdit = () => {
     const history = useHistory()
     const { id } = useParams()
 
-    const { loading, error, data: jsonData } = useReadGatewayQuery(id)
-    useCriticalNotification(error)
+    const { loading, error: loadError, data: jsonData } = useReadGatewayQuery(
+        id
+    )
 
     const [
         saveGenericGateway,
         { error: saveGenericGatewayError },
     ] = useUpdateGenericGatewayMutation()
-    useCriticalNotification(saveGenericGatewayError)
 
     const [
         saveBulkSMSGateway,
         { error: saveBulkSMSGatewayError },
     ] = useUpdateBulkSMSGatewayMutation()
-    useCriticalNotification(saveBulkSMSGatewayError)
 
     const [
         saveClickatellGateway,
         { error: saveClickatellGatewayError },
     ] = useUpdateClickatellGatewayMutation()
-    useCriticalNotification(saveClickatellGatewayError)
+
+    const saveError =
+        saveGenericGatewayError ||
+        saveBulkSMSGatewayError ||
+        saveClickatellGatewayError
+
+    if (loading) {
+        return (
+            <React.Fragment>
```

Comment:
Could be `<>` and `</>`. I find that easier to visually recognize than reading the actual name

Reply:
👍 Reason I got used to doing this is because this way you don't need your build tooling to support this specific notation (the short fragment notation needs babel 7+). But since our build tooling supports it I guess there's no reason to stick to the old version. I'll change it.

Reply:
Fixed in 026b198